library/build.gradle.kts

```diff
@@ -70,6 +70,10 @@ android {
     buildFeatures {
         buildConfig = true
     }
+    defaultConfig {
+        val consumerFile = file("consumer-rules.pro")
+        if (consumerFile.exists()) consumerProguardFiles(consumerFile)
```

Comment:
There is no need for this `if` check.
The file is there 😅 

Internally we only have this because most of our libraries doesn't need a consumer-proguard file 🙂